tools/hammer_loop.sh

```diff
@@ -29,6 +29,34 @@ if pgrep -fl -U "$(id -u)" "$cds"; then
     exit 1
 fi
 
+WORK_ROOT=${WORK_ROOT:-/tmp}
+TEST_ROOT="$WORK_ROOT/hammer_loop"
+if [[ -d "$TEST_ROOT" ]]; then
+    # Delete previous test data
+    rm -r "$TEST_ROOT"
+fi
+mkdir -p "$TEST_ROOT"
+if [[ $? -ne 0 ]]; then
+    echo "Failed to make test root $TEST_ROOT"
+    exit 1
+fi
+
+REGION_ROOT=${REGION_ROOT:-/var/tmp}
+MY_REGION_ROOT=${REGION_ROOT}/hammer_loop
+if [[ ! -d "$MY_REGION_ROOT" ]]; then
```

Comment:
Should this set of conditionals be organized in the same way as `TEST_ROOT` (delete old directory, then make new directory and check success)?  Right now, if `MY_REGION_ROOT` already exists, then we delete it and _don't_ recreate the directory.

(also applicable in many other scripts)

Reply:
Yeah, that's what I had locally, just failed to push it up.